### PR TITLE
TRUNK-3274, when checking for duplicate addresses ignore voided entries

### PR DESCRIPTION
**Pull request by cioan**
@mseaton , sorry for the previous request. My pull request is only one line of code. Thanks!

### CODE REVIEW

File: web/src/main/java/org/openmrs/web/controller/patient/ShortPatientFormValidator.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public void validate(Object obj, Errors errors) {
 			if (OpenmrsUtil.nullSafeEquals(possibleDuplicate.getId(), personAddress.getId()))
 				continue;
 			
-			if (!possibleDuplicate.isBlank() && !personAddress.isBlank()
+			if (!possibleDuplicate.isVoided() && !possibleDuplicate.isBlank() && !personAddress.isBlank()
```
Comment (dkayiwa) [inline web/src/main/java/org/openmrs/web/controller/patient/ShortPatientFormValidator.java]: How about a unit test? :)

Comment (cioan) [inline web/src/main/java/org/openmrs/web/controller/patient/ShortPatientFormValidator.java]: Hi @dkayiwa , thanks for the reminder. :-) Here is the unit test. Thank you for reviewing this pull request.

**Comment by dkayiwa** [comment]
@cioan do you mind squashing the commits into one?

**Comment by cioan** [comment]
All set. Thanks @dkayiwa !

**Pull request merged**